ql/src/test/org/apache/hadoop/hive/ql/TestTxnCommands.java

```diff
@@ -618,7 +618,13 @@ public void testMultipleInserts() throws Exception {
     dumpTableData(Table.ACIDTBL, 1, 1);
     List<String> rs1 = runStatementOnDriver("select a,b from " + Table.ACIDTBL + " order by a,b");
     Assert.assertEquals("Content didn't match after commit rs1", allData, rs1);
+    runStatementOnDriver("delete from " + Table.ACIDTBL + " where b = 2");
```

Comment:
This is a valid test, but I think the testMultipleInserts test for inserts, and this is test for deletes. Maybe create its' own test method named testDeleteOfInserts like testUpdateOfInserts?